cypress/support/fragments/settings/users/customFields.js

```diff
@@ -44,11 +44,22 @@ export default {
     // Wait for changes to be saved and reflected
     cy.wait(15000);
   },
-
+  createViaApi(field) {
```

Comment:
Why do you need this method? I can't find you using this.

Reply:
I delete this method, also the cy.getAdminToken(); but as I see it`s the same, I`ll do one more time!